src/Bindings/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ SET (SRCS
 
 SET (HDRS
 	Bindings.h
+	../ChannelManager.h
```

Comment:
Okay, this should not be here, the file belongs to the project in CMakeLists.txt up one level. However, it needs to be listed in `BINDING_DEPENDENCIES`